docs/guides/mlflow.rst

```diff
@@ -0,0 +1,22 @@
+MLFlow Utilities
+================
+
+Uploading Artifacts
+-------------------
+
+Artifacts must be written to a file before being uploaded to MLFlow. Rather than manually configuring an appropriate temporary directory for creating these files, we provide the ``get_temp_artifacts_dir()`` utility. This function can be used in any runtime and will return an appropriate directory.
+
+.. code-block:: python
+
+    from orquestra import sdk
+    from pathlib import Path
+    import mlflow
+
+    artifacts_dir: Path = sdk.mlflow.get_temp_artifacts_dir()
+
+    artifact_path = artifacts_dir / "final_state_dict.pickle"
+    with artifact_path.open("wb") as f:
+        pickle.dumps(my_model.state_dict())
+    mlflow.log_artifact(artifact_path)
```

Comment:
**suggestion**: have you considered turning this into a snippet within a test, similar to most of our other snippets? We could mock out `mlflow.log_artifact()`. This way this snippet won't get stale!

Reply:
Good idea, [66554b9](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/223/commits/66554b9f2d3d922a5c4828fc57b37b25a24c2356)